setup.py

```diff
@@ -24,6 +25,16 @@
         'pytest',
         'pyyaml',
         'peewee',
-        'monitorframe @ git+https://github.com/spacetelescope/monitor-framework#egg=monitorframe'
-    ]
-)
+        'monitorframe',
+        'tqdm'
+        # 'monitorframe @ git+https://github.com/spacetelescope/monitor-'
+        # 'framework#egg=monitorframe'
+        ],
+    dependency_links=[
```

Comment:
We've discussed this in person. 

This is not recommended for installing with `pip`. I think that it's also unnecessary as it has to do with a user's configuration with respect to GitHub rather than this project specifically. Because of that, I think we should leave this as is